addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -433,7 +433,7 @@ def _import_fill_invoice_allowance_charge(self, tree, invoice, journal, qty_fact
                 price_unit = float(amount_node.text) * charge_factor * qty_factor
 
             # get taxes
-            tax_xpath = './{*}TaxCategory/{*}Percent' if is_ubl else './{*}CategoryTradeTax/{*}RateApplicablePercent'
+            tax_xpath = './/{*}Percent' if is_ubl else './{*}CategoryTradeTax/{*}RateApplicablePercent'
```

Comment:
Same here. `'.//{*}Percent'` should be a fallback of `'./{*}TaxCategory/{*}Percent'`

Reply:
Actually there is no direct `Percent` child in [`AllowanceCharge`](http://www.datypic.com/sc/ubl21/e-cac_AllowanceCharge.html) node in UBL 2.1, my bad